content/influxdb/v2.0/monitor-alert/custom-checks.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Create custom checks
+seotitle: Custom checks
+description: >
+  Create custom checks with a Flux task.
+menu:
+  influxdb_2_0:
+    parent: Monitor & alert
+weight: 201
+v2.0/tags: [alerts, checks, tasks, Flux]
+---
+
+In the UI, you can create two kinds of [checks](/influxdb/v2.0/reference/glossary/#check):
+[`Threshold check`](/influxdb/v2.0/monitor-alert/checks/create/#threshold-check) and
+[`Deadman check`](/influxdb/v2.0/monitor-alert/checks/create/#deadman-check).
```

Comment:
to visually tie w line 19, we could format like this:
[`threshold` check](/influxdb/v2.0/monitor-alert/checks/create/#threshold-check) and
[`deadman` check](/influxdb/v2.0/monitor-alert/checks/create/#deadman-check).